src/components/text/index.js

```diff
@@ -35,7 +35,11 @@ export default class Text extends PureBaseComponent {
     /**
      * Custom highlight style for highlight string
      */
-    highlightStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.number])
+    highlightStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.number]),
+    /**
+     * Turn off accessibility for this view and its nested children
+     */
+    inaccessible: PropTypes.bool
```

Comment:
I thought we said we won't add it for now?

Reply:
Removed